src/Symfony/Component/Validator/CHANGELOG.md

```diff
@@ -16,7 +16,17 @@ CHANGELOG
  * added `PositiveOrZero` constraint
  * added `Negative` constraint
  * added `NegativeOrZero` constraint
-
+ * added `AbstractComposite` constraint
+ * deprecated `All` in favor of `Each` constraint
+ * deprecated `AllValidator` in favor of `EachValidator` validator
```

Comment:
the two deprecations need to be documented in the relevant upgrade files (`UPGRADE-4.4.md` and `UPGRADE-5.0.md`)